common/prepare-track/init.sh

```diff
@@ -325,7 +335,18 @@ function test_agent () {
 
     if [ "$connected" = true ]
     then
-        FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+        case "$INSTALL_WITH" in
+            helm)
+                FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            docker)
+                FOUND_COLLECTOR=`docker logs sysdig-agent 2>&1 | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            host)
+                FOUND_COLLECTOR=`echo 'journalctl -u dragent | grep -q "${CONNECTED_MSG}"'`
```

Comment:
I think this should be:
```suggestion
                FOUND_COLLECTOR=`echo 'journalctl -u dragent | grep "collector:" | head -n1 | awk '{print $NF}''`
```

Reply:
```suggestion
                FOUND_COLLECTOR=`grep -q "collector:" /opt/draios/logs/draios.log | head -n1 | awk '{print $NF}'`
```